recipes/graylog-forwarder/files/post-install

```diff
@@ -0,0 +1,73 @@
+#!/bin/sh
+
+set -e
+
+user="graylog-forwarder"
+group="graylog-forwarder"
+bindir="/usr/share/graylog-forwarder"
+datadir="/var/lib/graylog-forwarder"
+cfgdir="/etc/graylog/forwarder"
+
+# Flag to indicate if this is an upgrade
+upgrade="false"
+
+case "$1" in
+	# DEB based systems
+	configure)
+		if [ -z "$2" ]; then
+			upgrade="false"
+
+		else
+			upgrade="true"
+		fi
+		;;
+	abort-deconfigure|abort-upgrade|abort-remove)
+		;;
+	# RPM based systems
+	1)
+		# Installation
+		upgrade="false"
+		;;
+	2)
+		# Upgrade
+		upgrade="true"
+		;;
+	*)
+		echo "[ERROR] post-install script called with unknown argument: '$1'"
+		exit 1
+		;;
+esac
+
+#ensure files are owned by graylog-forwarder user
+chown -R "$user:$group" "$cfgdir"
+
+#ensure node-id is in the correct directory
+if [ -s "${bindir}/data/node-id" ]  && [ ! -s "${datadir}/data/node-id" ]; then
+	mv "${bindir}/data" "${datadir}/data"
```

Comment:
Let's make sure to only move the node-id file:
```suggestion
	mv "${bindir}/data/node-id" "${datadir}/data/node-id"
```

Reply:
Ok.